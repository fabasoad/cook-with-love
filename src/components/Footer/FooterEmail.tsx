import * as React from 'react'

export default function FooterEmail() {
  return (
    <div className="col">
      <i className="fa fa-envelope fa-2x"></i>
      <p>vanillacaramel88@gmail.com</p>
    </div>
  )
}
