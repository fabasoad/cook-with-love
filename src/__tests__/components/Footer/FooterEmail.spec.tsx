/// <reference types="jest" />
import { shallow } from 'enzyme'
import * as React from 'react'
import FooterEmail from '../../../components/Footer/FooterEmail'

describe(FooterEmail.name, () => {
  test('should render FooterEmail correctly', () =>
    expect(shallow(<FooterEmail />)).toMatchSnapshot()
  )
})
