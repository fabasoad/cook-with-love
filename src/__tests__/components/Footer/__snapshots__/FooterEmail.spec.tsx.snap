// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render FooterEmail correctly 1`] = `
<div
  className="col"
>
  <i
    className="fa fa-envelope fa-2x"
  />
  <p>
    vanillacaramel88@gmail.com
  </p>
</div>
`;
